src/Xamarin.Android.Build.Tasks/Xamarin.Android.TargetFrameworkInference.targets

```diff
@@ -0,0 +1,19 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Please place this file into [`src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android`](https://github.com/xamarin/xamarin-android/tree/master/src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android).

Reply:
ok, I can assume that the file will be deployed/installed in the same directory where the Xamarin.Android.CSharp.targets is located, right? @dellis1972 @jonpryor

Reply:
Yup, so long as as the [correct `%(Link)` metadata is provided](https://github.com/xamarin/xamarin-android/blob/master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets#L45-L49).